jenkins_jobs/build_host_os/script.sh

```diff
@@ -1,5 +1,5 @@
 # ISO 8601 date with nanoseconds precision
-TIMESTAMP=$(date --utc -Ins)
+TIMESTAMP=$(date --utc +'%Y-%m-%dT%H:%M:%S:%N+%Z')
```

Comment:
Do you know if yum has a problem with the `,`, the `+` or both? If it accepts `.`, we could use `+%Y-%m-%dT%H:%M:%S.%N+%Z` to keep following the standard. It would also be more close to Python's isoformat (e.g. `2017-02-06T17:05:19.278788`)